### PR TITLE
[CF-454] Use pbr library

### PR DESCRIPTION
**Pull request by SVilgelm**
Use setup.cfg file instead of setup.py

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Merge check failed
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3358/

**Comment by SVilgelm** [comment]
we can use `pbr` when https://review.openstack.org/#/c/311929/ is merged and new version of `pbr` is available

**Comment by MirantisWorkloadMobility** [comment]
Merge check failed
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3364/

File: setup.cfg [hunk 1]
```diff
@@ -9,3 +37,11 @@ with-xunit=1
...
 build-dir  = sphinx-build
 all_files  = 1
+
+[extras]
+docs =
+    sphinx
+    sphinx_rtd_theme
+tests =
+    oslotest
+    nose==1.3.7
```
Comment (roman-verchikov) [inline setup.cfg]: Let's remove these `extras`, we should not duplicate requirements in different places

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests failed
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3371/

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
can be merged

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**